### ISSUE TITLE
Basic pipeline script

### ISSUE THREAD

**Issue by dlevenstein** [buzsakilab/buzcode#345]
Should go from after recording to basic set of buzcode files

**Comment by dlevenstein** [comment]
add ipshita as well

**Comment by brendonw1** [comment]
Do we already have something like this?   I have my own for example.  Problem is everyone will want different things.   I suggest the following: create a script that pretty much offers all currently - available functions and then people can make their own copy and comment out certain functions.   But If we do it my way it can serve as an almost advertisement or communication about the current buzcode capaxities... which often people don’t know (I didn’t knew there was a bz_getwaveform).  I can submit my version if people like this idea and then others can modify

**Comment by AntonioFR8** [comment]
Yes, this is a good idea. Manu is working in a script like that but it will be good to compare it with yours. 
The idea we discussed was to have a general script that launch all the basic analysis from the raw data and that can be customize for each person.

**Comment by brendonw1** [comment]
I think it might be much faster to do it this way initially before putting in buzcode after initial agreement about elements:

Feel free to edit this:
https://docs.google.com/document/d/1oHySkezYWg9-6pq2KFLW7yrE0ARkIZOW1fKMDV5Up6k/edit?usp=sharing
